### PR TITLE
Add region tags to bigtable/hello sample.

### PR DESCRIPTION
**Pull request by tswast**
Also, change the sample to use sequential keys (with a disclaimer) to
match the Java sample. I had forgotten to add a sample usage to get a
specific row, so add that, too.

### CODE REVIEW

File: bigtable/hello/main.py [hunk 1]
```diff
@@ -27,13 +27,17 @@
...
 import argparse
-import uuid
 
 from gcloud import bigtable
 from gcloud.bigtable import happybase
 
 
 def main(project, cluster_id, zone, table_name):
+    # [START connecting_to_bigtable]
+    # project = "my-project-id"
```
Comment (jonparrott) [inline bigtable/hello/main.py]: These comments are unnecessary.

**Comment by jonparrott** [comment]
LGTM with small nit, but @waprin should review.

File: bigtable/hello/main.py (hunk 1 above)
Comment (lesv) [inline bigtable/hello/main.py]: I agree w/ Jon - rename project => projectId (or the Python convention), same for the others.

**Comment by lesv** [comment]
LGTM - w/ Jon's nit.

File: bigtable/hello/main.py (hunk 1 above)
Comment (tswast) [inline bigtable/hello/main.py]: Unnecessary comments removed and `project` renamed to `project_id`.

**Pull request merged**